contracts/src/access/ownable_two_step.rs

```diff
@@ -279,7 +279,7 @@ mod tests {
     }
```

Comment:
```suggestion
        assert_eq!(contract.sender(alice).pending_owner(), bob);
    }
```
noticed that this check is missing